### PR TITLE
Simplifies biggest func of Array extension

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by justJS** [comment]
I think this should wait for #41 otherwise it'll have to be done over again with Quadrilateral.

File: WeScan/Extensions/Array+Utils.swift [hunk 1]
```diff
@@ -12,15 +12,7 @@ extension Array where Element: CIRectangleFeature {
...
-        guard count > 1 else {
-            return first
-        }
-        
-        let biggestRectangle = self.max(by: { (rect1, rect2) -> Bool in
-            return rect1.perimeter() < rect2.perimeter()
-        })
-        
-        return biggestRectangle
+        return self.max { $0.perimeter() < $1.perimeter() }
```
Comment (AvdLee) [inline WeScan/Extensions/Array+Utils.swift]: `self` is probably not needed here

**Comment by AvdLee** [comment]
@justJS I think we're able to merge this in without any problems. We could solve any merge conflicts if they appear.

**Comment by valeriyvan** [comment]
>Would you mind writing a small test for it as well?
My intension was starting from writing tests by I failed to find a way to instantiate CIRectangleFeature.

> Also, are we sure that if count < 1, first is returned just like in the old code?
I am pretty sure max is correct for any number of elements, 0, 1, ... From Documentation:
Returns: The sequence’s maximum element if the sequence is not empty; otherwise, nil.

> self is probably not needed here
Not needed but is silences compiler warning. We have this warning without self:
Use of 'max' treated as a reference to instance method in protocol 'Sequence'
Use 'self.' to silence this warning
Use 'Swift.' to reference the global function

**Comment by justJS** [comment]
@AvdLee Well, #41 removes .biggest for CIRectangleFeature, because we moved everything to Quadrilateral. So yes, this will merge but it'll be useless.

**Comment by AvdLee** [comment]
@justJS makes sense. In that case, this code addition is indeed not needed. Hope you understand @valeriyvan, thanks for your contribution and time though!

**Comment by valeriyvan** [comment]
Will this be merged somehow?

**Comment by AvdLee** [comment]
@valeriyvan as pointed out in the above discussion, this code is no longer needed as it will be removed in an upcoming merge.

**Pull request closed without merging**